### PR TITLE
Replace Windows Server 2008 tokens

### PR DESCRIPTION
**Pull request by nxtn**

### CODE REVIEW

File: docs/framework/wcf/feature-details/unsupported-scenarios.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ For various reasons, Windows Communication Foundation (WCF) does not support som
  FIPS compliant AES encryption does not work in duplex callbacks under identification level impersonation.  
   
 ### CNG/KSP Certificates  
- *Cryptography API: Next Generation (CNG)* is the long-term replacement for the CryptoAPI. This API is available in unmanaged code on Windows Vista,  [!INCLUDE[lserver](../../../../includes/lserver-md.md)] and later Windows versions.  
+ *Cryptography API: Next Generation (CNG)* is the long-term replacement for the CryptoAPI. This API is available in unmanaged code on Windows Vista,  Windows Server 2008 and later Windows versions.
```
Comment (gewarren) [inline docs/framework/wcf/feature-details/unsupported-scenarios.md]:
```suggestion
 *Cryptography API: Next Generation (CNG)* is the long-term replacement for the CryptoAPI. This API is available in unmanaged code on Windows Vista, Windows Server 2008, and later Windows versions.  
```

File: docs/framework/wcf/feature-details/how-to-replace-the-wcf-url-reservation-with-a-restricted-reservation.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ A URL reservation allows you to restrict who can receive messages from a URL or
   
  As part of the default operating system configuration, Windows Communication Foundation (WCF) creates a globally accessible reservation for port 80 to enable all users to run applications that use a dual HTTP binding for duplex communication. Because the ACL on this reservation is for everyone, administrators cannot explicitly allow or disallow permission to listen on a URL or set of URLs. This topic explains how to delete this reservation and how to re-create the reservation with a restricted ACL.  
   
- On Windows Vista or [!INCLUDE[lserver](../../../../includes/lserver-md.md)] you can view all of the HTTP URL reservations from an elevated command prompt by typing `netsh http show urlacl`.  The following example shows what a WCF URL reservation should resemble.  
+ On Windows Vista or Windows Server 2008 you can view all of the HTTP URL reservations from an elevated command prompt by typing `netsh http show urlacl`.  The following example shows what a WCF URL reservation should resemble.
```
Comment (gewarren) [inline docs/framework/wcf/feature-details/how-to-replace-the-wcf-url-reservation-with-a-restricted-reservation.md]:
```suggestion
On Windows Vista or Windows Server 2008, you can view all of the HTTP URL reservations from an elevated command prompt by entering `netsh http show urlacl`. The following example shows what a WCF URL reservation should resemble:
```

**Review by gewarren — COMMENTED** [review]
@NextTurn Thanks for replacing this token. I left a couple unrelated suggestions.

**Review by tdykstra — APPROVED** [review]
@NextTurn Thanks!

**Pull request merged**